src/components/organisms/post/PostDetailSection.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <section class="flex" id="posts-detail-section">
+    <div class="posts-detail-container">
+      <!-- load header section -->
+      <q-card>
+        <q-card-section>
+          <q-icon :name="matArrowBack" size="25px" @click="back"/>
+          <div class="posts-detail-header">
+            <PostDetailHeader :loading="loading" :post="post" />
+          </div>
+        </q-card-section>
+      </q-card>
+      <!-- load image -->
+      <div v-bind:class="[mediaQueryPlatform == 'desktop' ? 'posts-detail-image floated': '']">
+        <PostDetailFeaturedImage :loading="loading" :post="post" :platform="mediaQueryPlatform" />
+      </div>
+      <!-- load content -->
+      <div class="row">
+        <div class="col-md-8 col-sm-9 col-xs-12">
+          <PostDetailContent :loading="loading" :post="post" />
+          <div v-if="needLogin" class="flex justify-center items-center need-login">
+            <q-btn
+              size="sm"
+              color="primary"
+              label="Please login to see full content"
+              @click="() => $router.push('/login')" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import PostDetailFeaturedImage from "src/components/molecules/post/PostDetailFeaturedImage.vue";
+import PostDetailHeader from "src/components/molecules/post/PostDetailHeader.vue";
+import PostDetailContent from "src/components/molecules/post/PostDetailContent.vue";
+import PostBlock from "src/lib/domain/entities/post_block";
+import { mapActions, mapState } from "vuex";
+import { matArrowBack } from '@quasar/extras/material-icons'
+
+export default {
+  components: {
+    PostDetailFeaturedImage,
+    PostDetailContent,
+    PostDetailHeader
+  },
+  name: "PostDetailSection",
+  async mounted() {
+    await this.getPostBySlug(this.$route.params.slug);
+    if (!this.post) return this.$router.push("/404");
+  },
+  computed: {
+    ...mapState("post", ["post", "loading", "error"]),
+    blocks() {
+      if (!this.post.content || this.post.content.blocks.length === 0) return [];
+      return this.post.content.blocks.map((block) => PostBlock.fromJson(block));
+    },
+    needLogin() {
+      if (!this.post) return false;
+      return this.post.needLogin;
+    },
+  },
+  methods: {
+    ...mapActions("post", ["getPostBySlug"]),
+    back() {
+      this.$router.back();
+    },
+  },
+  created() {
+    this.matArrowBack = matArrowBack
+    // get media query size
+    this.screenWidth = window.innerWidth;
```

Comment:
We can use the Quasar built in utils to works with breakproints, you can find it inside Style & Identity menu on Quasar v1 documentation. References:
- https://v1.quasar.dev/style/breakpoints
- https://v1.quasar.dev/style/visibility
- https://v1.quasar.dev/style/sass-scss-variables
- https://v1.quasar.dev/style/other-helper-classes